src/components/CustomerQuota/NoQuotaCard.tsx

```diff
@@ -36,6 +41,12 @@ const styles = StyleSheet.create({
   },
   itemDetail: {
     fontSize: fontSize(-1)
+  },
+  appealButton: {
```

Comment:
Perhaps `appealButtonText`? Thought this referred to the entire `TouchableOpacity` when I first read it.

Reply:
Done